tests/mocks/sampler_black-box.py

```diff
@@ -29,12 +32,23 @@
 # If it got everything correctly, it will produce a dummy terminal output and
 # a) particle_lists.oscar in the output directory if the optional argument is not given
 # b) nothing if the BLACK_BOX_FAIL environment variable is set to "true"
+#
+# Use the MOCK_HADRON_SAMPLER_VERSION environment variable to set the sampler version
+# behavior which is mimicking the different real sampler versions.
 
 import sys
 import os
 import time
 import random
 import numpy as np
+from packaging.version import Version
+
+sampler_version = os.environ.get("MOCK_HADRON_SAMPLER_VERSION", "3.1.1")
```

Comment:
Does it make sense to have as the default the old version, as in the future, we will have the new call?

Reply:
Ideally, I would have preferred not to have a default one. Just to say: The test must decide how to test. But I did not find time for that and I also think that it is not so bad to have the old one. As you say, we will more and more often use the newer versions. However, we do not set any requirement, so it is not bad to check by default that everything still works with the older sampler. 🤔

Reply:
Sure, it works like this too. Maybe it will make future people appreciate the new way to call the sampler more :)